src/habit/Habit.js

```diff
@@ -1,4 +1,6 @@
 import React from "react";
+import { useState, useEffect } from "react";
+import moment from "moment";
 import { useState } from "react";
```

Comment:
I think CircleCI threw an error because of the repeat import of useState on line 2 and line 4